tests/caasp/transactional_update.pm

```diff
@@ -63,7 +63,10 @@ sub run {
     trup_call "ptf install" . rpmver('security');
     check_reboot_changes;
     check_package 'in';
-    my $snap = script_output "snapper list | tail -1 | awk '{print \$3}'";
+
+    # Find snapshot number for rollback
+    my $f = is_caasp('kubic') ? 1 : 3;
```

Comment:
We already got a method to find column and not to rely on the hardcoded values: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/snapper_create.pm#L18 Would be great if you could reuse it.

Reply:
Thanks, did not know about it - I am testing changed PR now

Reply:
We don't have an easy way to know about that. I know only because I've reviewed that PR. And in general we tend to implement similar thing multiple times.

Reply:
as a heads up, get_last_snap_number seems to have a bug :) 

```
bash -eox pipefail -c "snapper list | head -n1"; echo $?                                                                                                                                               
+ snapper list
+ head -n1
Type   | #   | Pre # | Date                     | User | Cleanup | Description             | Userdata                                                                                                             
141
```

Reply:
Bug is sometimes there, sometimes not :(

Reply:
We could use `proceed_on_failure` argument in script_output call, but if it's even sporadic, I'm not sure if there is easy way out =(

Reply:
Love this... I am thinking about going back to hardcoded values :(
![screenshot from 2018-11-05 14-12-18](https://user-images.githubusercontent.com/538604/48000136-eb623200-e104-11e8-9e1b-e24c086e0ec5.png)

Reply:
pipestatus 141 won, I reverted to hardcoded values & started last local run

Reply:
FYI - pipestatus 141 is 'SIGPIPE'. It's entirely possible that it's a valid response given the piping magic going on.. maybe the pipefail magic is a little too magic?

Reply:
@kravciak Have you filed a bug? I do understand that investigation will take a while here, but it will already fail in the code we currently have =(
@foursixnine Have you created ticket for it already? So at least we could put a comment to reuse fixed test code here as well?

Reply:
@rwx788 - not sure it's a bug, you can try on you machine too:
```bash
:~ sudo snapper list | head -n1; echo ${PIPESTATUS[*]}
Type   | #   | Pre # | Date        | User | Cleanup | Description  | Userdata     
0 0
:~ sudo snapper list | head -n1; echo ${PIPESTATUS[*]}
Type   | #   | Pre # | Date       | User | Cleanup | Description  | Userdata     
141 0
```

Reply:
We have done same with @foursixnine today in the morning, but still I don't understand why we've started to get the error which is thrown e.g. when pipe doesn't exist anymore. I have TW, so it just shows that I can reproduce same behavior as in test =(

Reply:
workaround: `... | tail -n +1 | head -1 ...` did it for me

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6126

Reply:
Me and Santi tried with using || true and it seemed to help as well. It's rather a question if no users have similar setup and if we should apply pipefail only for specific cases and not for e.g. script_output in particular. Let's see.

Reply:
@rwx788 https://progress.opensuse.org/issues/43364 :)